packages/protocol/test/stability/reserve.ts

```diff
@@ -283,6 +285,48 @@ contract('Reserve', (accounts: string[]) => {
     })
   })
 
+  describe('addExchangeSpender & removeExchangeSpender', () => {
```

Comment:
I know this will feel pedantic, but we generally group our tests by method, and therefore I would recommend to do here the same. Additionally, I would add a test with the above suggested view call

Reply:
done!